docs/content/workers/worker-entity.md

```diff
@@ -1,20 +1,19 @@
 [//]: # (Doc of docs reference 15.2a)
 
 # Workers: Worker entity
-_This document relates to both *[GameObject-MonoBehaviour and  ECS workflows](../intro-workflows-spos-entities.md)_
+_This document relates to both *[GameObject-MonoBehaviour and  ECS workflows]({{urlRoot}}/content/intro-workflows-spos-entities.md)_
 
 
-Before reading this document, see the documentation on [workers in the GDK](./workers-in-the-gdk.md).
+Before reading this document, see the documentation on [workers in the GDK]({{urlRoot}}/content/workers/workers-in-the-gdk.md).
 
-Each of the workers in your project must have exactly one [ECS entity](../glossary.md#unity-ecs-entity) in its [worker-ECS world](./workers-in-the-gdk.md# workers-and-ecs-worlds) at any point in time. To uniquely identify the worker entity of your current worker, the worker entity has the `WorkerEntityTag` component attached to it.
+Each of the workers in your project must have exactly one [ECS entity]({{urlRoot}}/content/glossary.md#unity-ecs-entity) in its [worker-ECS world]({{urlRoot}}/content/workers/workers-in-the-gdk.md#workers-and-ecs-worlds) at any point in time. To uniquely identify the worker entity of your current worker, the worker entity has the `WorkerEntityTag` component attached to it.
 
 The worker’s worker entity performs certain tasks:
 
-
-  * send and receive [commands](../glossary.md#commands) before the worker has checked out any SpatialOS entities.
-  * register changes to the state of the Runtime connection (that is whether the worker is connected to the [Runtime](../glossary.md#spatialos-runtime) or not) by filtering for the following [temporary components](temporary-components.md):
- 	* `OnConnected`: the worker just connected to the SpatialOS [Runtime](../glossary.md#spatialos-runtime).
- 	* `OnDisconnected`: the worker just disconnected from the SpatialOS [Runtime](../glossary.md#spatialos-runtime). This is an `ISharedComponentData` and stores the reason for the disconnection as a `string`.
+  * send and receive [commands (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#command) before the worker has checked out any SpatialOS entities.
+  * register changes to the state of the Runtime connection (that is whether the worker is connected to the [Runtime]({{urlRoot}}/content/glossary.md#spatialos-runtime) or not) by filtering for the following [temporary components]({{urlRoot}}/content/ecs/temporary-components.md):
+     * `OnConnected`: the worker just connected to the SpatialOS [Runtime]({{urlRoot}}/content/glossary.md#spatialos-runtime).
+     * `OnDisconnected`: the worker just disconnected from the SpatialOS [Runtime]({{urlRoot}}/content/glossary.md#spatialos-runtime). This is an `ISharedComponentData` and stores the reason for the disconnection as a `string`.
```

Comment:
linking multiple times for the same term, but should be fine for the first version